justfile

```diff
@@ -112,12 +112,22 @@ fix: devenv
 # Run the dev project
 run:
     #!/usr/bin/env bash
-    port=$(echo $SERVER_HOST | awk -F: '{print $3}' | tr -d / )
+    port=$(echo $RELEASE_HOST | awk -F: '{print $3}' | tr -d / )
     $BIN/uvicorn hatch.app:app --reload --port $port
 
 
 token WORKSPACE="workspace":
-    $BIN/python dev-token.py "{{ WORKSPACE }}"
+    @$BIN/python dev-token.py "{{ WORKSPACE }}"
+
+
+get-index WORKSPACE="workspace":
+    @curl -s http://localhost:8001/workspace/{{ WORKSPACE }}/current -H "Authorization: $({{ just_executable() }} token {{ WORKSPACE }})"
+
+
+get-file FILE WORKSPACE="workspace":
+    #!/bin/bash
+    url=$({{ just_executable() }} get-index {{ WORKSPACE }} | jq -r '.files[] | select(.name="{{ FILE }}").url')
+    curl -s $url -H "Authorization: $({{ just_executable() }} token {{ WORKSPACE }})"
```

Comment:
A comment to explain what these are for would be nice.

Reply:
Good point, I missed that.

I think in hindsight, this would be better implemented in python, I will add it to the dev-token script